### PR TITLE
Update: Dashboards to ember 3.25

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
Update dashboards to latest

## Proposed Changes
- clean up deprecation workflow

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/dashboards/addon/templates/components/common-actions/buttonize.hbs [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<button type="button" class="btn">
```
Comment (jkusa) [inline packages/dashboards/addon/templates/components/common-actions/buttonize.hbs:2]: 🎉

File: packages/reports/addon/components/report-actions/multiple-format-export.js [hunk 2]
```diff
@@ -122,16 +123,24 @@ export default class MultipleFormatExport extends Component {
...
   /**
    * @action close
    */
   @action
   close(dropdown) {
-    dropdown.actions.close();
+    this.closeTimer = later(() => {
+      this.closeTimer = null;
+      dropdown.actions.close();
+    }, 300);
```
Comment (jkusa) [inline packages/reports/addon/components/report-actions/multiple-format-export.js:143]: What's happening here?

**Review by jkusa — APPROVED** [review]
Just the one question

File: packages/reports/addon/components/report-actions/multiple-format-export.js (hunk 2 above)
Comment (kevinhinterlong) [reply packages/reports/addon/components/report-actions/multiple-format-export.js:143]: This is moving https://github.com/yavin-dev/framework/pull/1343 forward so it's replacing the `closeDelay` behavior from BasicDropdownHover

Comment (jkusa) [reply packages/reports/addon/components/report-actions/multiple-format-export.js:143]: got it

**Pull request merged**